app/src/main/java/io/ably/example/androidpushexample/MainActivity.kt

```diff
@@ -820,8 +835,9 @@ class MainActivity : AppCompatActivity() {
 			R.id.action_reset_local_device -> resetLocalDevice()
 			R.id.action_get_activation_state -> getActivationState()
 			R.id.action_reset_activation_state -> resetActivationState()
-            R.id.action_push_subscribe_client -> pushSubscribeClient()
-            R.id.action_push_unsubscribe_client -> pushUnsubscribeClient()
+			R.id.action_push_subscribe_client -> pushSubscribeClient()
+			R.id.action_push_unsubscribe_client -> pushUnsubscribeClient()
+			R.id.action_regenerate_client_id -> regenerateClientId()
```

Comment:
Maybe add a `action_get_client_id` so we can display the current `clientId` ?

Reply:
Pushed this change. 
https://github.com/ably/push-example-android/pull/9/commits/1b0e86e34a0ba5438e948737a2d86b934126afcf